<launch>

	<arg name="mission_index" default="1" doc="index of the desired mission to run" />
	<arg name="landing_lat" default="37.4224444" doc="GPS latitude of the landing location in decimal degrees" />
	<arg name="landing_lon" default="-122.1760917" doc="GPS longitude of the landing location in decimal degrees" />

	<node pkg="aa241x_mission" type="aa241x_mission_node" name="aa241x_mission_node" output="screen">
		<param name="mission_index" value="$(arg mission_index)" />
		<param name="mission_file" value="$(find aa241x_mission)/missions/id$(arg mission_index).mission" type="str" />
		<param name="landing_lat" value="$(arg landing_lat)" type="double" />
		<param name="landing_lon" value="$(arg landing_lon)" type="double" />
	</node>

	<!-- a logging node for data for the TAs to confirm scores -->
	<node pkg="rosbag" type="record" name="scoring_bag"
		args="-o $(env HOME)/rosbags/scoring_bag
				/mission_state
				/measurement
				/person_found
				/mavros/state
				/mavros/global_position/global
				/geodetic_based_lake_lag_pose
				/mavros/local_position/pose
				/mavros/altitude
				/mavros/time_reference
				/rosout" />

</launch>